<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shapeworks Viewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout">
    <property name="margin">
     <number>8</number>
    </property>
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QVTKWidget" name="vtkWidget" native="true"/>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mainMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Parameter_File"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionView_Samples"/>
    <addaction name="actionView_Mean"/>
    <addaction name="actionView_Median"/>
    <addaction name="actionView_PCA_Modes"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionShow_Glyphs"/>
    <addaction name="actionColor_Scheme"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuActions"/>
   <addaction name="menuOptions"/>
  </widget>
  <action name="actionSub1">
   <property name="text">
    <string>Sub1</string>
   </property>
  </action>
  <action name="actionSub2">
   <property name="text">
    <string>Sub2</string>
   </property>
  </action>
  <action name="actionSub3">
   <property name="text">
    <string>Sub3</string>
   </property>
  </action>
  <action name="actionLoad_Parameter_File">
   <property name="text">
    <string>Load Parameter File</string>
   </property>
   <property name="statusTip">
    <string>Load Parameter File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit Viewer</string>
   </property>
   <property name="statusTip">
    <string>Exit Viewer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionView_Samples">
   <property name="text">
    <string>View Samples</string>
   </property>
  </action>
  <action name="actionView_Mean">
   <property name="text">
    <string>View Mean</string>
   </property>
  </action>
  <action name="actionView_Median">
   <property name="text">
    <string>View Median</string>
   </property>
  </action>
  <action name="actionView_PCA_Modes">
   <property name="text">
    <string>View PCA Modes</string>
   </property>
  </action>
  <action name="actionShow_Glyphs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show Glyphs</string>
   </property>
  </action>
  <action name="actionColor_Scheme">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Color Scheme</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends></extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
